heartbeat/mariadb.in

```diff
@@ -247,6 +247,39 @@ The port on which the Promoted MariaDB instance is listening.
 <content type="string" default="${OCF_RESKEY_replication_port_default}" />
 </parameter>
 
+<parameter name="replication_require_ssl" unique="0" required="0">
+<longdesc lang="en">
+Enables SSL connection to local MySQL service for replication user.
+i.e. if REQUIRE SSL for replication user in MySQL set, this should be set to "true".
+</longdesc>
+<shortdesc lang="en">MySQL replication require ssl</shortdesc>
+<content type="string" default="${OCF_RESKEY_replication_require_ssl_default}" />
```

Comment:
You have to define the _default parameters and assign the value from them when not defined, see example:
https://github.com/ClusterLabs/resource-agents/blob/main/heartbeat/awseip#L48-L58

Reply:
That is OK if the parameters are also used by the mysql agent.

Reply:
That was the IDEA behind this. The only thing I had to change in mysql-common.sh was the handlng of "MYSQL_OPTIONS_LOCAL_SSL_OPTIONS" because mariadb is using --ssl and mysql --ssl-mode=REQUIRED.